kubectl-cloudflow/docker/inspect_test.go

```diff
@@ -0,0 +1,59 @@
+// +build integration
+
+package docker
+
+import (
+	"fmt"
+	"gotest.tools/assert"
+	"testing"
+	"time"
+)
+
+var imageNameToTest = "docker.io/lightbend/spark-aggregation:134-d0ec286-dirty"
+
+func Test_inspect(t *testing.T) {
+	ins := NewInspectOptions()
+
+	start := time.Now()
+	output, err := ins.InspectRemoteDockerImage("docker.io/lightbend/spark-aggregation:134-d0ec286-dirty")
```

Comment:
Add some tests?
- inspect a not existing remote docker image
- inspect an image that you do not have credentials for
(Would also prefer gingko tests here ;-)

Reply:
it is possible.